extensions/interactions/ItemSelectionInput/ItemSelectionInput.js

```diff
@@ -27,30 +27,56 @@ oppia.directive('oppiaInteractiveItemSelectionInput', [
       scope: {},
       templateUrl: 'interaction/ItemSelectionInput',
       controller: ['$scope', '$attrs', function($scope, $attrs) {
-        $scope.items = oppiaHtmlEscaper.escapedJsonToObj($attrs.itemsWithValue);
-        var allowableSelections = $attrs.selectionCountWithValue;
+        $scope.choices = oppiaHtmlEscaper.escapedJsonToObj($attrs.choicesWithValue);
+        var maxAllowableSelectionCount = $attrs.maxAllowableSelectionCountWithValue;
+        var minAllowableSelectionCount = $attrs.minAllowableSelectionCountWithValue;
         $scope.studentSelections = {};
 
-        for (var i = 0; i < $scope.items.length; i++) {
-          $scope.studentSelections[$scope.items[i]] = false;
+        if (maxAllowableSelectionCount == 1) {
+          $scope.isCheckbox = false;
```

Comment:
Prefer

  $scope.displayCheckboxes

(since otherwise you're saying whether the $scope is a checkbox, which doesn't really make sense). Also, you can replace all this with:

   $scope.displayCheckboxes = (maxAllowableSelectionCount > 1);